src/components/Coins/Coin.css

```diff
@@ -0,0 +1,52 @@
+.coin-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.info {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  gap: 30px;
+  align-items: center;
+  padding: 10px 30px;
+  box-sizing: border-box;
+  margin-bottom: 20px;
+}
+
+.info img {
+  border-radius: 50%;
+  height: 150px;
```

Comment:
- [ ] Please use the correct font families: `gill sans` for the body and title and `lato` as control font.

![image](https://user-images.githubusercontent.com/68796232/197370490-c254bda0-dc1b-4e11-ab24-960663db94df.png)